docs/vhlo.md

```diff
@@ -133,33 +133,28 @@ negative unit tests of the feature, as well as compatibility unit tests.
 
 Compatibility unit testing involves updating [stablehlo_legalize_to_vhlo.mlir](https://github.com/openxla/stablehlo/blob/main/stablehlo/tests/vhlo/stablehlo_legalize_to_vhlo.mlir)
 to ensure that StableHLO round trips with the latest version of VHLO, as well
-as any additional forward or backward compatibility tests required.
+as any additional forward or backward compatibility tests required. For example,
+if adding a new op at version `X` with `Y = X - 1`, add a test file like
+`vhlo_to_version_downgrade_invalid.0_Y_0.mlir` that shows the op is unsupported
+before version `X`. If adding a new version of an op, add a test file like
+`vhlo_to_version_downgrade.0_Y_0.mlir` that shows that the op can be downgraded
+successfully.
 
-A few examples:
-
-* Backward compatibility, positive tests: [vhlo_to_version_upgrade.mlir](https://github.com/openxla/stablehlo/blob/6886b59f6cd4369674e7e3beff61301c145176e2/stablehlo/tests/vhlo_to_version_upgrade.mlir#L2)
-* Forward compatibility, positive tests: [vhlo_to_version_downgrade.mlir](https://github.com/openxla/stablehlo/blob/6886b59f6cd4369674e7e3beff61301c145176e2/stablehlo/tests/vhlo_to_version_downgrade.mlir#L1)
-* Forward compatibility, negative tests: [vhlo_to_version_downgrade_invalid.0_9_0.mlir](https://github.com/openxla/stablehlo/blob/main/stablehlo/tests/vhlo/vhlo_to_version_downgrade_invalid.0_9_0.mlir)
```

Comment:
Should we keep these around? Or update to point at a recent invalid downgrade test?

Reply:
Added a pointer to a recent test. Thanks!